src/status_im2/contexts/wallet/collectible/view.cljs

```diff
@@ -110,21 +111,43 @@
         :subtitle      network-name
         :subtitle-type :network}]]]))
 
+(defn collectible-actions-sheet
+  []
+  [quo/action-drawer
+   [[{:icon                :i/messages
+      :accessibility-label :share-opensea-link
+      :label               (i18n/label :t/share-opensea-link)}
+     {:icon                :i/link
+      :accessibility-label :view-on-eth
+      :label               (i18n/label :t/view-on-eth)}
+     {:icon                :i/download
+      :accessibility-label :save-image-to-photos
+      :label               (i18n/label :t/save-image-to-photos)}
+     {:icon                :i/copy
+      :accessibility-label :copy-all-details
+      :label               (i18n/label :t/copy-all-details)}
+     {:icon                :i/share
+      :accessibility-label :share-details
+      :label               (i18n/label :t/share-details)}]]])
+
 (defn view
   []
-  (let [collectible                  (rf/sub [:wallet/last-collectible-details])
+  (let [collectible                       (rf/sub [:wallet/last-collectible-details])
         {:keys [collectible-data preview-url
-                collection-data]}    collectible
-        {:keys [traits description]} collectible-data
-        chain-id                     (rf/sub [:wallet/last-collectible-chain-id])]
+                collection-data]}         collectible
+        {:keys [traits name description]} collectible-data
+        chain-id                          (rf/sub [:wallet/last-collectible-chain-id])]
     [scroll-page/scroll-page
      {:navigate-back? true
       :height         148
       :page-nav-props {:type        :title-description
                        :title       name
                        :description description
                        :right-side  [{:icon-name :i/options
-                                      :on-press  #(js/alert "pressed")}]
+                                      :on-press  #(rf/dispatch
+                                                   [:show-bottom-sheet
+                                                    {:content collectible-actions-sheet
+                                                     :theme   nil}])}]
```

Comment:
why we passing theme as `nil` here btw? 🤔

Reply:
Oh, thanks for noticing!) That was temporary